fusor/model.py

```diff
@@ -53,90 +100,305 @@ class TranscriptComponent(BaseModel):
     _validate_transcript = \
         validator('transcript', allow_reuse=True)(check_curie)
 
-
-class CriticalDomain(BaseModel):
-    """Define CriticalDomain class"""
-
-    status: StrictStr
-    name: str
-    id: str
-    gene: GeneDescriptor
+    class Config:
+        """Configure class."""
+
+        @staticmethod
+        def schema_extra(schema, _):
+            """Provide example"""
+            if 'title' in schema.keys():
+                schema.pop('title', None)
+            for prop in schema.get('properties', {}).values():
+                prop.pop('title', None)
+            schema['example'] = {
+                'component_type': 'transcript_segment',
+                'transcript': 'NM_152263.3',
+                'exon_start': 1,
+                'exon_start_offset': 0,
+                'exon_end': 8,
+                'exon_end_offset': 0,
+                'gene': 'TPM3',
```

Comment:
We have gene set as a GeneDescriptor, but in here and I believe in the example file, it's a string. Unsure what it actually should be, but we should stay consistent.

Reply:
used to be a gene, should be a GeneDescriptor